opentelemetry-zipkin/CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## vNext
 
 - Bump msrv to 1.75.0.
+- **Breaking** The `opentelemetry_zipkin::new_pipeline()` interface is now replaced with `opentelemetry_zipkin::ZipkinExporter::builder()`.
```

Comment:
can you modify the PR description or the changelog itself to show before/after to help migration?

Reply:
please have a look, if the example is ok.